src/main/kotlin/org/jetbrains/research/tasktracker/config/content/TaskContentConfig.kt

```diff
@@ -3,17 +3,52 @@ package org.jetbrains.research.tasktracker.config.content
 import kotlinx.serialization.Serializable
 import org.jetbrains.research.tasktracker.config.BaseConfig
 import org.jetbrains.research.tasktracker.config.YamlConfigLoadStrategy
+import org.jetbrains.research.tasktracker.models.Extension
+import org.jetbrains.research.tasktracker.tracking.task.SourceSet
+import org.jetbrains.research.tasktracker.tracking.task.Task
+import org.jetbrains.research.tasktracker.tracking.task.TaskFile
 import java.io.File
 
 // TODO should it be a data class?
```

Comment:
Yes, it should) And for building tasks, it is better to create two separate classes with factory patterns, which can build tasks of task files by configs.

Reply:
fixed